docs/application/dotnet/api/overview.md

```diff
@@ -1,21 +1,21 @@
 
-- **.NET Standard API**, which implements the .NET Base Class library, allows you to use the well known C# language base class libraries and features, such as collections, threading, file I/O, and LINQ.
-- **Xamarin.Forms**, which allows you to efficiently build a user interface from standard components in C# or XAML.
-- **TizenFX API**,
-   which allows you to access platform-specific features not covered by the generic .NET and Xamarin.Forms features,  such as system information  and status, battery status, sensor date, and account and connectivity services.  For the full specification, see the <a href="tizenfx/index.html" target="api">TizenFX API Reference</a>
+- **.NET Standard API** implements the .NET Base Class library, and allows you to use the well-known C# language base class libraries and features, such as collections, threading, file input/output (I/O), and Language Integrated Query (LINQ).
+- **Xamarin.Forms** allows you to efficiently build a user interface from standard components in C# or XAML.
+- **TizenFX API** allows you to access platform-specific features not covered by the generic .NET and Xamarin.Forms features, such as system information and status, battery status, sensor date, and account and connectivity services.  
+  For the full specification, see the [TizenFX API Reference]( https://github.com/jkpu/tizen-docs/blob/b49532622a70fd10dec59657fe9d531cea5ab6af/docs/application/dotnet/api/tizenfx/index.html).
 
 ## .NET Standard API
 
-One of the major components of .NET Core is the .NET Standard. The .NET APIs provided by Tizen .NET follow the .NET Standard 2.0. The column titled "2.0" in the [official list](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) of supported CoreFX APIs lists all the available .NET APIs.
+One of the major components of .NET Core is the .NET Standard. The .NET APIs provided by Tizen .NET follows the .NET Standard 2.0. The column titled as 2.0 in the [official list](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) of the supported CoreFX APIs lists all the available .NET APIs.
 
 ## Xamarin.Forms
 
 [Xamarin.Forms](https://developer.xamarin.com/guides/xamarin-forms/getting-started/) provides cross-platform APIs, which allow you to create user interfaces that can be shared across platforms. The Tizen.NET Visual Studio extension enables Tizen support for Xamarin.Forms.
 
-You can efficiently build your Tizen .NET application UI and its supporting logic using the Xamarin.Forms APIs. Extended details for these APIs are available on the [Xamarin Web site](https://developer.xamarin.com/api/namespace/Xamarin.Forms/).
+You can efficiently build your Tizen .NET application user interface and its supporting logic using the Xamarin.Forms APIs. Extended details for these APIs are available on the [Xamarin Web site](https://developer.xamarin.com/api/namespace/Xamarin.Forms/).
```

Comment:
UI is acceptable. **UI** instead of **user interface** is more used in technical documents.